src/main/java/fi/metatavu/dcfb/server/rest/ItemsApiImpl.java

```diff
@@ -87,6 +115,7 @@ public Response createItem(Item payload) throws Exception {
     Long amount = payload.getAmount();
     String unit = payload.getUnit();
     UUID modifier = getLoggerUserId();
+    boolean visibilityLimited = payload.isVisibilityLimited() == null ? false : payload.isVisibilityLimited();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the literal "false" boolean value. ([squid:S1125](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1125))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=15&issues=AWStCknRIUupByjQEY1I&open=AWStCknRIUupByjQEY1I)

[ISSUE_KEY]: AWStCknRIUupByjQEY1I